tests/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+set(
+    sources
+    0001-multiobj.c
```

Comment:
> "generated build system cannot know when to ask CMake to regenerate."

What does that mean in practice?
When is the generated build system asking CMake to generate?